ha/core/health_monitor/monitor_rules_manager.py

```diff
@@ -97,7 +98,8 @@ def evaluate(self, event: HealthEvent) -> list:
             list: actions configured for the rule
         """
         val = []
-        key = self._prepare_key(event.resource_type, event.event_type)
+        functional_type = event.specific_info.get("type")
```

Comment:
any constant is defined?
as this needs to be used by the event sender also

Reply:
Modified to refer it from const. Please check.